### PR TITLE
[CC-46&48: Bugfix] update error message of password & remove back button in each step

### PR DESCRIPTION
**Pull request by Cameron933**
### Descriptions

- Link to Jira ticket here: 
  - https://leonhard.atlassian.net/browse/CC-46
  - https://leonhard.atlassian.net/browse/CC-48
- Link to Figma: None

### Provide screenshots or videos that the function is working.
image
image

### Provide screenshots of no warning, or error in the console.
image

- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket, (i.e `[CC-xx: Feature] Create pull request template`).
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] No unnecessary spaces between lines of code.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Check you named your code properly.
- [x] Remove unused comments.

### CODE REVIEW

**Review by VanderDeng — APPROVED** [review]

**Comment by NorrisWu0** [comment]
Run through the pr checklist again.

**Comment by NorrisWu0** [comment]
Screenshotting the test coverage doesn't tells me anything of whether this bug is fixed or not.

**Comment by NorrisWu0** [comment]
Remember to assign yourself in the pr.

**Comment by Cameron933** [comment]
> Run through the pr checklist again.

File: apps/galaxy/src/modules/auth/SignUpForm/CompanyInfoPage.tsx [hunk 1]
```diff
@@ -42,58 +42,50 @@ const CompanyInfoPage = (props: {
...
+          errorMessage={formState.errors.companyPostcode?.message}
+        />
+        <FormInput
+          {...businessAddress}
+          {...register("businessAddress")}
+          errorMessage={formState.errors.businessAddress?.message}
+        />
+      </Stack>
+      <Flex direction="column" gap="16px">
+        <Stack marginTop="24px" direction={["column", "row"]} justifyContent="stretch">
```
Comment (NorrisWu0) [inline apps/galaxy/src/modules/auth/SignUpForm/CompanyInfoPage.tsx:82]: Do you really need `<Stack>` here if there is only one button now?

File: apps/galaxy/src/modules/auth/SignUpForm/StaffInfoPage.tsx [hunk 2]
```diff
@@ -7,12 +7,10 @@ import useSignUp from "@src/services/signup/useSignUp";
 import { Dispatch, SetStateAction } from "react";
 
 const StaffInfoPage = (props: {
```
Comment (NorrisWu0) [inline apps/galaxy/src/modules/auth/SignUpForm/StaffInfoPage.tsx:9]: You can deconstruct data here if you are not reusing `props` in other area

**Review by NorrisWu0 — APPROVED** [review]
LGTM, but still left two comments for you to consider.

File: apps/galaxy/src/modules/auth/SignUpForm/CompanyInfoPage.tsx (hunk 1 above)
Comment (Cameron933) [reply apps/galaxy/src/modules/auth/SignUpForm/CompanyInfoPage.tsx:82]: I think keeping <Stack> here would be fine. The reason for this is that it is more appropriate to use a global layout than to hard code the height of the button in the component.
image
image

Also, I removed the direction property as there is now only one button.

Comment (NorrisWu0) [reply apps/galaxy/src/modules/auth/SignUpForm/CompanyInfoPage.tsx:82]: I don't see how this is related to `<GlobalLyaout>` and it is fine to have a or a set of fixed height for the buttons.

File: apps/galaxy/src/modules/auth/SignUpForm/StaffInfoPage.tsx (hunk 2 above)
Comment (Cameron933) [reply apps/galaxy/src/modules/auth/SignUpForm/StaffInfoPage.tsx:9]: Do you mean this:
image

**Pull request merged**